finding_video_links.md

```diff
@@ -58,4 +58,15 @@ You pretty much only have 3 options when that happens:
 
 1. Try to use a fake / no captcha token. Some sites actually doesn't check that the captcha token is valid.
 2. Use Webview or some kind of browser in the background to load the site in your stead.
-3. Pray it's a captcha without payload, then it's possible to get the captcha key without a browser: [Code example](https://github.com/LagradOst/CloudStream-3/blob/ccb38542f4b5685e511824a975bf16190011c222/app/src/main/java/com/lagradost/cloudstream3/MainAPI.kt#L132-L181)
+3. Pray it's a captcha without payload, then it's possible to get the captcha key without a browser:
+Here is a proof of concept code example of how you can get a captcha token programmatically (this can vary for various websites):
```

Comment:
Please document the script a little :pray: 
What is main_page, what is provider_id?